src/libraries/Microsoft.Extensions.Hosting/src/HostBuilder.cs

```diff
@@ -131,28 +134,27 @@ public IHost Build()
 
             // REVIEW: If we want to raise more events outside of these calls then we will need to
             // stash this in a field.
-            using var diagnosticListener = new DiagnosticListener("Microsoft.Extensions.Hosting");
-            const string hostBuildingEventName = "HostBuilding";
-            const string hostBuiltEventName = "HostBuilt";
+            using DiagnosticListener diagnosticListener = LogHostBuilding(this);
 
-            if (diagnosticListener.IsEnabled() && diagnosticListener.IsEnabled(hostBuildingEventName))
-            {
-                Write(diagnosticListener, hostBuildingEventName, this);
-            }
+            InitializeHostConfiguration();
```

Comment:
Why are all these methods being renamed?

Reply:
I was going for a pattern where the "Initialize" methods are the instance methods that initialize fields with the values retuned by the static "Create" methods. I can revert if this is too much churn.

Reply:
ok, got it. I think this is a good approach, no need to revert.